### PR TITLE
Add/release notes 1.1.0

### LINKED ISSUE #1121: Add Release Notes and Some Documents for Agora Chat V1.1.0
**Business case**: How will this issue help our customers.

**Subject Matter Expert**: Who the doc team can ask technical questions.

**Reviewers**:
  - *Technical*: 
  - *Language*: Iain
  - *PM*: only if essential

**Scope**: The products and documents to be updated. If you are not sure, leave it to Iain. 

**Acceptance criteria**: Criteria for this issue to be approved.

**Additional valuable information**: Anything else that will help with this issue.

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
- Resolves #1121

### CODE REVIEW

File: shared/chat-sdk/client-api/messages/send-receive-messages/project-implementation/ios.mdx [hunk 1]
```diff
@@ -20,6 +20,19 @@ message.chatType = AgoraChatTypeGroupChat;
                                              completion:nil];
 ```
 
+### Set message priority
```
Comment (atovpeko) [inline shared/chat-sdk/client-api/messages/send-receive-messages/project-implementation/ios.mdx:23]: Please apply my suggestion for the same intro in Android

File: shared/chat-sdk/client-api/messages/send-receive-messages/project-implementation/web.mdx [hunk 2]
```diff
@@ -28,6 +28,30 @@ function sendTextMessage() {
 }
 ```
 
+### Set message priority
+
```
Comment (atovpeko) [inline shared/chat-sdk/client-api/messages/send-receive-messages/project-implementation/web.mdx:32]: Please apply my suggestion for the same intro in Android

**Review by atovpeko — CHANGES_REQUESTED** [review]
Please:
1. Address my comments
2. Ping the CN writer for updated RNs (to remove the `XXX`)
Thank you

**Comment by Kishan-Dhakan** [comment]
@atovpeko I've resolved the comments and pinged the CN writer on the original issue.

**Review by atovpeko — APPROVED** [review]
I am approving this hoping the `XXX` blanks will be fixed before this is merged

**Comment by Kishan-Dhakan** [comment]
@atovpeko merging this after discussing with Iain, when the new RNs are available I'll update it in a separate PR.

**Pull request merged**